src/components/body/main/HobbySection.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Box, Chip, Grid, Typography,
+} from '@mui/material';
+
+import { Section } from './Section';
+
+export function HobbySection(): JSX.Element {
+  const text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse pulvinar '
+    + 'lobortis nisl et porttitor. Maecenas in felis neque. Suspendisse et scelerisque lectus. '
+    + 'Maecenas pretium felis ac massa interdum auctor.';
```

Comment:
As previous and please put this variable to the additional file in the directory where current component placed